ts/features/idpay/onboarding/xstate/services.ts

```diff
@@ -1,17 +1,45 @@
 /* eslint-disable no-underscore-dangle */
 import * as E from "fp-ts/lib/Either";
-import * as O from "fp-ts/lib/Option";
 import { pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
 import { PreferredLanguage } from "../../../../../definitions/backend/PreferredLanguage";
-import { InitiativeDto } from "../../../../../definitions/idpay/onboarding/InitiativeDto";
-import { StatusEnum } from "../../../../../definitions/idpay/onboarding/OnboardingStatusDTO";
+import { InitiativeInfoDTO } from "../../../../../definitions/idpay/onboarding/InitiativeInfoDTO";
+import { StatusEnum as OnboardingStatusEnum } from "../../../../../definitions/idpay/onboarding/OnboardingStatusDTO";
+import { DetailsEnum as PrerequisitesErrorDetailsEnum } from "../../../../../definitions/idpay/onboarding/PrerequisitesErrorDTO";
 import { RequiredCriteriaDTO } from "../../../../../definitions/idpay/onboarding/RequiredCriteriaDTO";
 import { SelfConsentDTO } from "../../../../../definitions/idpay/onboarding/SelfConsentDTO";
 import { OnboardingClient } from "../api/client";
 import { OnboardingFailureEnum } from "./failure";
 import { Context } from "./machine";
 import { getBoolRequiredCriteriaFromContext } from "./selectors";
 
+// prettier-ignore
```

Comment:
Not needed

Reply:
This is to avoid text wrapping by prettify, without it readability is reduced with no improvement. 
You can see the result below:

`const onboardingStatusToFailure: Record<
  OnboardingStatusEnum,
  O.Option<OnboardingFailureEnum>
> = {
  [OnboardingStatusEnum.ELIGIBILE_KO]: O.some(
    OnboardingFailureEnum.NOT_ELIGIBLE
  ),
  [OnboardingStatusEnum.ONBOARDING_KO]: O.some(
    OnboardingFailureEnum.NO_REQUIREMENTS
  ),
  [OnboardingStatusEnum.ONBOARDING_OK]: O.some(OnboardingFailureEnum.ONBOARDED),
  [OnboardingStatusEnum.UNSUBSCRIBED]: O.some(
    OnboardingFailureEnum.UNSUBSCRIBED
  ),
  [OnboardingStatusEnum.ELIGIBLE]: O.some(OnboardingFailureEnum.ON_EVALUATION),
  [OnboardingStatusEnum.ON_EVALUATION]: O.some(
    OnboardingFailureEnum.ON_EVALUATION
  ),
  [OnboardingStatusEnum.ACCEPTED_TC]: O.none, // Onboarding started but not yet completed, no failure
  [OnboardingStatusEnum.INVITED]: O.none // Whitelisted CF, no failure
};

const prerequisitesErrorToFailure: Record<
  PrerequisitesErrorDetailsEnum,
  OnboardingFailureEnum
> = {
  [PrerequisitesErrorDetailsEnum.BUDGET_TERMINATED]:
    OnboardingFailureEnum.NO_BUDGET,
  [PrerequisitesErrorDetailsEnum.INITIATIVE_END]: OnboardingFailureEnum.ENDED,
  [PrerequisitesErrorDetailsEnum.INITIATIVE_NOT_STARTED]:
    OnboardingFailureEnum.NOT_STARTED,
  [PrerequisitesErrorDetailsEnum.INITIATIVE_SUSPENDED]:
    OnboardingFailureEnum.SUSPENDED,
  [PrerequisitesErrorDetailsEnum.GENERIC_ERROR]: OnboardingFailureEnum.GENERIC
};`